roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -22,18 +22,21 @@ openshift_prometheus_pvc_name: prometheus
 openshift_prometheus_pvc_size: "{{ openshift_prometheus_storage_volume_size | default('10Gi') }}"
 openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
 openshift_prometheus_pvc_pv_selector: "{{ openshift_prometheus_storage_labels | default({}) }}"
+openshift_prometheus_sc_name: "{{ openshift_prometheus_storage_class | default(None) }}"
```

Comment:
@jmencak Please add documentation for the added variables here:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/README.md#storage-related-variables

and the examples from the description here:
https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L618

Reply:
I'll add some information into README.md.  As for hosts.example, I don't see the "pvc" method of installation described there at all and it also contains variables (e.g. openshift_prometheus_storage_kind) that (no longer) exist.  I'd say it needs to be cleaned up first.

Reply:
@jmencak hosts example has been cleaned up:
https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example#L626-L692

Please add this example there

Reply:
@zgalor example added as requested.